example/terrafrom/api.tf

```diff
@@ -61,8 +61,14 @@ resource "aws_api_gateway_integration" "menu" {
 # Send Message
 ###################################################################################################
 
-resource "aws_api_gateway_resource" "send_message" {
+resource "aws_api_gateway_resource" "send_message_bot" {
```

Comment:
Shouldn't this be updated to be renamed to Webhooks?

Reply:
That file needs a lot of love lol